assets/runtime/functions

```diff
@@ -329,13 +329,20 @@ redmine_configure_memcached() {
 
   if [[ ${MEMCACHE_ENABLED} == true ]]; then
     echo "Configuring redmine::memcached..."
-    install_template ${REDMINE_USER}: redmine/additional_environment.rb ${REDMINE_MEMCACHED_CONFIG} 0644
-    update_template ${REDMINE_MEMCACHED_CONFIG} \
-      MEMCACHE_HOST \
-      MEMCACHE_PORT
+    {
+      echo "config.gem 'dalli'"
+      echo "config.action_controller.perform_caching  = true"
+      echo "config.cache_classes = true"
+      echo "config.cache_store = :mem_cache_store, ${MEMCACHE_HOST}:${MEMCACHE_PORT}"
+    } >> "${REDMINE_MEMCACHED_CONFIG}"
   fi
 }
 
+redmine_configure_logger() {
+  echo "Configuring redmine::logger..."
+  echo "config.log_level = :${LOGGER_LEVEL:-info}" >> "${REDMINE_MEMCACHED_CONFIG}"
```

Comment:
Also put the LOGGER_LEVEL default in env-defaults so its consistent with the other settings.

Reply:
Done